antora-preview.sh

```diff
@@ -96,4 +96,4 @@ echo "| More info here: https://github.com/vshn/antora-preview#livereload   |"
 echo "|_____________________________________________________________________|"
 echo ""
 caddy start
-signalListener guard --no-interactions --group=documentation
+signalListener guard --no-interactions -w antora/src public
```

Comment:
This should probably use `$ANTORA_PATH`, since that's the user-configurable path to where in the repo the antora docs live:

```suggestion
signalListener guard --no-interactions -w "antora/${ANTORA_PATH}" public
```

Reply:
Yes, this change is required. Merging! Thanks Simon!